src/aiperf/dataset/generator/video.py

```diff
@@ -330,6 +335,7 @@ def _create_video_with_pipes(self, frames: list[Image.Image]) -> str:
             # Add format-specific options for streaming/pipe output
             if self.config.format == VideoFormat.MP4:
                 # For pipes, we need frag_keyframe and empty_moov for non-seekable output
+                # Note: This path is now skipped for MP4 (see _create_video_with_ffmpeg)
```

Comment:
If this path is never used for MP4s, I think it would make sense to remove this path from the function.